packages/editor/src/dataviews/actions/trash-post.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * WordPress dependencies
+ */
+import { trash } from '@wordpress/icons';
+import { useDispatch } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	__experimentalText as Text,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import type { Action } from '@wordpress/dataviews';
+
+/**
+ * Internal dependencies
+ */
+import { getItemTitle, isTemplateOrTemplatePart } from './utils';
+import type { CoreDataError, PostWithPermissions } from '../types';
+
+const trashPost: Action< PostWithPermissions > = {
+	id: 'move-to-trash',
+	label: __( 'Move to Trash' ),
```

Comment:
I know this wasn't introduced in this PR, but:

I thought we use sentence case nowadays? i.e. `Move to trash` with lowercase t.